File: scala/src/main/scala/io/digitalstream/library/Collections.scala
/**
  * Created by sebby on 9/25/16.
  */
package io.digitalstream.library

object Collections {

  def run(): Unit = {
    val names = Seq("alpha", "beta", "gamma")
    names
      .filter(_.startsWith("a"))
      .map(_.toUpperCase())
      .foreach(println)
  }
}

File: kotlin/src/main/kotlin/io/digitalstream/library/Collections.kt
/**
 * Created by sebby on 9/25/16.
 */
package io.digitalstream.library

object Collections {

  fun run(): Unit {
    val names = listOf("alpha", "beta", "gamma")
    names
        .filter { it.startsWith("a") }
        .map(String::toUpperCase)
        .forEach(::println)
  }
}
